scripts/component_generation/main.go

```diff
@@ -0,0 +1,195 @@
+package main
```

Comment:
`/scripts` - where is the logic for self-registering apps and filters located currently?